manual/src/refman/extensions/tail_mod_cons.etex

```diff
@@ -0,0 +1,548 @@
+(Introduced in OCaml 4.14)
+
+Note: this feature is considered experimental, and its interface may
+evolve, with user feedback, in the next few releases of the language.
+
+Consider the simple way of implementing the "List.map" function:
+\begin{verbatim}
+let rec map f li =
```

Comment:
The stdlib docs consistently (I think!) use `l` for lists. `li` reads oddly to me - could we have either `l` or `xs`/`ls`?

Reply:
I don't like `l` which looks too close to `|` or `1` for comfort on many fonts, but I forgot that `i` is a pronunciation minefield in English. Let's go with `l` for consistency, thanks.